resources/js/components/contents/Dashboard.js

```diff
@@ -1,10 +1,31 @@
 import React from "react";
 
+import { Container, Grid, Paper, Typography } from "@mui/material";
+
 const Dashboard = () => {
   return (
-    <div>
-      <div>Dashboard</div>
-    </div>
+    <React.Fragment>
+      <Container>
+        <Grid container>
+          <Grid item lg={4} md={4} sm={4} xs={12}>
+            <Typography component="h5" variant="h5" sx={{ mb: 5 }}>
+              Dashboard
+            </Typography>
+          </Grid>
+          <Grid item lg={8} md={8} sm={8} xs={12}>
+            {/* {props.requestErrorMessage !== "" ? (
+              <AlertContent
+                isError={props.requestError}
+                message={props.requestErrorMessage}
+              />
+            ) : null} */}
```

Comment:
remove if not needed, but if will be used later, dont mind this comment

Reply:
I have removed unused code. This is resolved ma'am.